.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "6"
+# after_success:
+  # - sh set_tags.sh
+script:
+  - npm run rebel
+deploy:
+  provider: releases
+  file: "./_build/src/rebel"
+  skip_cleanup: true
+  api_key:
+    secure: "HL0Z7JB++6nM4AQlRBm3jFTw69IddGqga31XirfOPe2jecCiE8cVKb05OLjIrv0VyKXh/UQmnhZQ/uipCIcjpGlXYFV8WMUkoCHiYOPnLP6hawj0F+H965VVukuQ6xPA8vdoK+9UDJW8tvSdIe3Ejb78JQxfPUE8+6GqqsWpwaHKxf4Lz/sLGcjVtzSJt7k4qFFs3OJrGA+SLcnsKTrE8/9IUgve1T+MY35R3M7sCKKjVUWGn3BOxwVDxZrrbPd5jnIE2uHAXimu6ssTFGDjMSvBpWgJi5LpbVPnV+GvXLXteqviqAIOCM7R2N2E2GlUKSYq4J2ntx3CTIwRwkH85dz+8TaD+mDal+QA+dKEf6rpgYB/u/sFmUjvqgU3pBk7QxjDm6+/zO1t9oblEUCWWAaD/0bCn85frKTSlap6j2nPJTuBKXog4QIH4MOgvAFBbPm7kFn7Z6qlAzUFZKT4cDg5fKUhNcYY9jq924OY7RSXzEAKaIWYubGpmHj0TmASK+ishZh/SGfhvXdnEzQm9UWdVP9Fj3NmUFOZryf9nTxn7fbizLkSEsrdwMbZ6CFmk+s5B66jvXKamh2RCtPAsOiN/1J6Wxzq3uAVDRX8l50vWtzYNfnoGXLRLKRxCda3Fkyxt5ZidDFWiQ7GzXIETbL12XnU98gMAMvg0tpYIbg="
```

Comment:
This is my github token to push releases to my fork of rebel. I am still testing. I got linux releases working. 😄